### PR TITLE
Adding tmux-cargo plugin

### PR DESCRIPTION
**Pull request by idevtier**
[tmux-cargo](https://github.com/idevtier/tmux-cargo)

Plugin for running [cargo](https://github.com/rust-lang/cargo) commands in tmux.

### CODE REVIEW

**Comment by idevtier** [comment]
Moved plugin in alphabetical order.

File: README.md [hunk 1]
```diff
@@ -96,3 +97,4 @@ Pro tip: watch this repository to get notified about new plugins.
 - [tmux-yank](https://github.com/tmux-plugins/tmux-yank) - Plugin for copying
   to system clipboard. Works on MacOS, Linux and Cygwin.
 - [tpm](https://github.com/tmux-plugins/tpm) - Tmux plugin manager.
+
```
Comment (bruno-) [inline README.md:100]: Can you please remove this last line? Everything else is ok and I'll merge afterwards.

Comment (idevtier) [reply README.md]: My bad, I'll fix that, thank you

**Comment by idevtier** [comment]
Hey, I've removed last line, can you merge it please

**Pull request merged**